metaflow/plugins/kfp/constants.py

```diff
@@ -5,4 +5,4 @@
 DEFAULT_RUN_NAME = "run_mf_on_kfp"
 DEFAULT_EXPERIMENT_NAME = "mf-on-kfp-experiments"
 
-BASE_IMAGE = "hsezhiyan/metaflow-zillow:1.1"
+BASE_IMAGE = "hsezhiyan/metaflow-zillow:1.2"
```

Comment:
@talebzeghmi @cloudw this new image includes `PyTest`, which is needed for the `merge_artifacts_flow.py` test file.